Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -105,31 +106,178 @@ is ready to operate:
 
 Note, in above helm configuration the ``kubeProxyReplacement`` has been set to
 ``strict`` mode. This means that the Cilium agent will bail out in case the
-underlying Linux kernel support is missing. Without explicitly specifying a
-``kubeProxyReplacement`` option, helm uses ``kubeProxyReplacement`` with ``probe``
-by default which would automatically disable a subset of the features to implement
-the kube-proxy replacement instead of bailing out if the kernel support is
-missing. This makes the assumption that Cilium's BPF kube-proxy replacement would
-co-exist with kube-proxy on the system to optimize Kubernetes services. Given
-we've used kubeadm to deploy a kube-proxy-free setup, the ``strict`` mode
-has been used instead.
+underlying Linux kernel support is missing.
+
+Without explicitly specifying a ``kubeProxyReplacement`` option, helm uses
+``kubeProxyReplacement`` with ``probe`` by default which would automatically
+disable a subset of the features to implement the kube-proxy replacement instead
+of bailing out if the kernel support is missing. This makes the assumption that
+Cilium's BPF kube-proxy replacement would co-exist with kube-proxy on the system
+to optimize Kubernetes services. Given we've used kubeadm to explicitly deploy
+a kube-proxy-free setup, the ``strict`` mode has been used instead to ensure
+that we do not rely on a (non-existing) fallback.
+
+Cilium's BPF kube-proxy replacement is supported in direct routing as well as in
+tunneling mode.
+
+Direct Server Return (DSR) and other NodePort Settings
+######################################################
 
 When running Cilium's BPF kube-proxy replacement, by default, a NodePort or
 ExternalIPs service will be accessible through the IP address of a native device
 which has the default route on the host. To change the device, set its name in
 the ``global.nodePort.device`` helm option.
 
 In addition, thanks to the :ref:`host-services` feature, the NodePort service can
-be accessed from a host or a Pod within a cluster via it's public, cilium_host
-device or loopback address, e.g. ``127.0.0.1:NODE_PORT``.
+be accessed by default from a host or a Pod within a cluster via it's public,
+cilium_host device or loopback address, e.g. ``127.0.0.1:NODE_PORT``.
+
+By default, Cilium's BPF NodePort implementation operates in SNAT mode. That is,
```

Comment:
I'd mention that in the SNAT mode a client's source IP addr is not preserved, while in the DSR mode it is.